### PR TITLE
added advanced logging config for audit

### PR DESCRIPTION
**Pull request by thefactremains**
#### Summary

https://github.com/mattermost/mattermost-server/pull/15076
@wiggin77 I'm not sure if the sample files apply the same way. If not, please let me know if you suggest replacing them.

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -4726,6 +4726,25 @@ This setting can be left as default unless you are seeing audit write failures i
...
+
+- Multiple local file targets
+- Multiple syslogs
+- Multiple TCP sockets
+
+Allow any combination of local file, syslog, and TCP socket targets. 
+
+File target supports rotation and compression triggered by size and/or duration. Syslog target supports local and remote syslog servers, with or without TLS transport. TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+
+This feature's ``config.json`` setting is ``ExperimentalAuditSettings.AdvancedLoggingConfig`` which can contain a filespec to another config file, a database DSN, or JSON. Options are outlined in this txt file: `Log Settings Options <https://github.com/mattermost/docs/files/5066579/Log.Settings.Options.txt>`_. Sample config: `Advanced Logging Options Sample.json.zip <https://github.com/mattermost/docs/files/5066597/Advanced.Logging.Options.Sample.json.zip>`_.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:4746]: Should we mention that both links are file downloads and not an actual page?

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -4726,6 +4726,25 @@ This setting can be left as default unless you are seeing audit write failures i
 | This feature's ``config.json`` setting is ``"FileMaxQueueSize": 1000`` with numerical input.                                                                       |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Advanaced Audit Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*Available in Enterprise Edition E20*
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:4731]: This needs a line break between the header format and "*Available in...*"

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -4726,6 +4726,25 @@ This setting can be left as default unless you are seeing audit write failures i
 | This feature's ``config.json`` setting is ``"FileMaxQueueSize": 1000`` with numerical input.                                                                       |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Advanaced Audit Logging Configuration
```
Comment (amyblais) [inline source/administration/config-settings.rst:4729]:
```suggestion
Advanced Audit Logging Configuration
```

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -4726,6 +4726,25 @@ This setting can be left as default unless you are seeing audit write failures i
 | This feature's ``config.json`` setting is ``"FileMaxQueueSize": 1000`` with numerical input.                                                                       |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Advanced Audit Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*Available in Enterprise Edition E20*
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:4731]:
```suggestion

*Available in Enterprise Edition E20*
```

File: source/administration/config-settings.rst (hunk 1 above)
Comment (justinegeffen) [reply source/administration/config-settings.rst:4747]: ^^ Non-blocking comment.

**Review by justinegeffen — APPROVED** [review]

**Review by amyblais — APPROVED** [review]

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -4726,6 +4724,26 @@ This setting can be left as default unless you are seeing audit write failures i
...
+
+- Multiple local file targets
+- Multiple syslogs
+- Multiple TCP sockets
+
+Allow any combination of local file, syslog, and TCP socket targets. 
+
+File target supports rotation and compression triggered by size and/or duration. Syslog target supports local and remote syslog servers, with or without TLS transport. TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+
+This feature's ``config.json`` setting is ``ExperimentalAuditSettings.AdvancedLoggingConfig`` which can contain a filespec to another config file, a database DSN, or JSON. Options are outlined in this txt file: `Log Settings Options <https://github.com/mattermost/docs/files/5066579/Log.Settings.Options.txt>`_. Sample config: `Advanced Logging Options Sample.json.zip <https://github.com/mattermost/docs/files/5066597/Advanced.Logging.Options.Sample.json.zip>`_.
```
Comment (wiggin77) [inline source/administration/config-settings.rst:4745]: These sample files are valid for auditing as well as logging.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

File: source/administration/config-settings.rst (hunk 1 above)
Comment (thefactremains) [reply source/administration/config-settings.rst:4745]: I'm ok with leaving them this way. I copied these directly from the advanced logging page.

**Pull request merged**